service/src/main/java/tds/assessment/services/impl/AssessmentAssembler.java

```diff
@@ -84,12 +84,13 @@ static void assemble(final Assessment assessment,
                     .ifPresent(s -> s.getForms().add(form));
             }
         }
-        
+
         // ADAPTIVE SEGMENTS - Update the segment with data based on its selection algorithm
         for (Segment segment : assessment.getSegments()) {
             segment.setStrands(strands.stream()
                 .filter(strand -> strand.getSegmentKey().equals(segment.getKey()))
                 .collect(Collectors.toSet()));
+
             segment.setItems(items.stream()
```

Comment:
Ok, I tried to remove setting the items for fixed form since all the items are on the form, but that change ripples.  I'll remove the extra item setting in a separate PR because it isn't critical to remove the blocker for scoring.